runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process.json

```diff
@@ -48,6 +48,7 @@
     "Process.Run.ManageProcessForm": "FORMULARZ STARTOWY",
     "Process.Run.ScheduleAttended.Message": "Harmonogram jest dostępny tylko dla nienadzorowanych botów",
     "Process.Configure.Attended.Schedule.Message": "Bot dodany do harmonogramu nie może być nadzorowany",
+    "Process.MainView.Link.RunBoticsTutorial": "RunBotics - Samouczek",
```

Comment:
here too, lets remove the dash and change the order of words. `Samouczek RunBotics` sounds more polish